Open_Science_New.html

```diff
@@ -474,134 +516,24 @@ <h2><font color="0f4078">Real-Time Query Results</font></h2>
-              <p class="fragment current-visible"><font size="9" color="blue">1 Terabyte dataset</font></p>
-              <p class="fragment current-visible"><font size="9" color="blue"><a href="https://bigbrain.humanbrainproject.org/#!{'layers':{' grey value: ':{'type':'image'_'source':'precomputed://https://neuroglancer.humanbrainproject.org/precomputed/BigBrainRelease.2015/8bit'_'transform':[[1_0_0_-70666600]_[0_1_0_-70000000]_[0_0_1_-58777700]_[0_0_0_1]]}_' tissue type: ':{'type':'segmentation'_'source':'precomputed://https://neuroglancer.humanbrainproject.org/precomputed/BigBrainRelease.2015/classif'_'selectedAlpha':0_'segments':['0'_'100']_'transform':[[1_0_0_-70666600]_[0_1_0_-72910000]_[0_0_1_-58777700]_[0_0_0_1]]}}_'navigation':{'pose':{'position':{'voxelSize':[21166.666015625_20000_21166.666015625]_'voxelCoordinates':[-3338.57958984375_856.3231811523438_-853.1627807617188]}}_'zoomFactor':1400706.647149517}_'perspectiveOrientation':[0.3140767216682434_-0.7418519854545593_0.4988985061645508_-0.3195493221282959]_'perspectiveZoom':1922235.5293810747}" target="_blank"><font color="f27c30"><small>Click to launch the Neuroglancer Viewer!</small></font></a></font></p>
             </section>
-	    
-	    <!-- CCNA  -->
-	    <section data-transition="concave">
-	      <table class="reveal">
-		<h2>CCNA</h2>
-		<tr>
-		  <td width="70%" style="vertical-align: top">
-		    <img src="images/CCNA.png">
-		  </td>
-		  <td width="30%" style="vertical-align: top">
-		    <p><small>
-			<ul>
-			  <li>$35 million</li><br>
-			  <li>CIHR funded</li><br>
-			  <li>Pan Canadian Consortium</li><br>
-			  <li>Acquisition in June, 2016</li><br>
-			  <li>35 collections sites</li><br>
-			  <li>LORIS powered</li><br>
-			</ul>
-		    </small></p>
-		</td>
-	      </tr></table>
-	    </section>
-
-	    <!-- Phantom -->
-	  <section>
-	    <img src="images/Phantom.png">
-	  </section>
-	    
-	    <!-- QPN -->
-	    <section data-transition="fade">
-	      <img src="images/QPN.png" width="110%">
-	    </section>
-	    
-	    <!-- PreventAD -->
-	  <section>
-	    <img src="images/PreventAD.png" width="87%">
-	  </section>
 
-	  <!-- Cuban Brain Mapping -->
-	  <section data-transition="convex">
-	    <a href="" target="_blank"><img src="images/Cuban_brain_mapping.png" width="120%"></a>
-	  </section>
-	    <!-- ABIDE -->
-	    <section data-transition="fade">
-	      <table class="reveal">
-		<h2>ABIDE</h2>
+        <!-- Brainhack -->
+	  <section data-transition="fade">
+	    <table class="reveal">
+	      <h2>Hackathons</h2>
 	      <tr>
-		<td width="45%" style="vertical-align: middle">
-		  <img src="images/ABIDE_paper.png">
+		<td width="75%" style="vertical-align: middle">
+		  <img src="images/C-BASE.png">
 		</td>
-		<td width="55%" style="vertical-align: middle">
-		  <img src="images/ABIDE_preprocessed.png">
+		<td width="25%" style="vertical-align: middle">
+		  <p> Important value of hackathons to data sharing initiatives</p>
```

Comment:
- [ ] instead
Vital environments for collaboration and exchange of ideas, tools and solutions for data sharing initiatives
- [ ] it looks too narrowly constricted - try the width